helpdesk/templates/helpdesk/ticket.html

```diff
@@ -159,7 +159,14 @@ <h5 class="mb-1">{{ followup.title|escape|num_to_link }}</h5>
                 <dd><input type='text' name='title' value='{{ ticket.title|escape }}' /></dd>
 
                 <dt><label for='id_owner'>{% trans "Owner" %}</label></dt>
-                <dd><select id='id_owner' name='owner'><option value='0'>{% trans "Unassign" %}</option>{% for u in active_users %}{% if u.id == ticket.assigned_to.id %}<option value='{{ u.id }}' selected>{{ u }}</option>{% else %}<option value='{{ u.id }}'>{{ u }}</option>{% endif %}{% endfor %}</select></dd>
+                <dd>
+                    <select id='id_owner' name='owner'>
+                        <option value='0'>{% trans "Unassign" %}</option>
```

Comment:
The use of the word "Unassign" indicates an action wheras "Unassigned" indicates a state so maybe as part of this PR rename it to "Unassigned".

This would make it aligned wit hthe nomenclature on the owner filter:
[https://github.com/django-helpdesk/django-helpdesk/blob/main/helpdesk/templates/helpdesk/filters/owner.html#L12](https://github.com/django-helpdesk/django-helpdesk/blob/main/helpdesk/templates/helpdesk/filters/owner.html#L12)